apps/design-land/src/app/card/card.component.html

```diff
@@ -0,0 +1,9 @@
+<h1>Daff Card</h1>
+
+<h2>Card</h2>
+<daff-card>
+  <img daffCardImage src="/assets/card/uber-logo.svg" />
+  <h4 daffCardTitle>Title</h4>
+  <p daffCardBody>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```

Comment:
I don't like this API. I don't want to have to add `daffCardBody` to every element I want inside a `card`.

Ideally, I think that `daffCardBody` should be `implicit` with an API like:

```
<daff-card>
  <img daffCardImage src="/assets/card/uber-logo.svg" />
  <h4 daffCardTitle>Title</h4>
  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
</daff-card>
```

makes more sense.

Reply:
Resolved in 22430c3